### PR TITLE
Add documentation on supported compression algorithms

### PR DESCRIPTION
**Pull request by lfrancke**
# Description

This takes the guesswork out of it for our customers.
We tell them which algorithms are supported and which aren't.

### CODE REVIEW

File: docs/modules/hbase/pages/usage-guide/compression.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+= Compression support
+
+Our images of Apache HBase support compressed column families.
+We support the following compression algorithms for all HBase 2.4 versions:
+
+- GZip
+- Snappy
```
Comment (maltesander) [inline docs/modules/hbase/pages/usage-guide/compression.adoc:7]: Either the operator tests are crap or we do support more :) ?
https://github.com/stackabletech/hbase-operator/blob/18c98a0824d6f46befbe3af88b5a5bd2e6f44d7a/tests/templates/kuttl/smoke/test-hbase.py#L82

Comment (lfrancke) [reply docs/modules/hbase/pages/usage-guide/compression.adoc:7]: The other ones should not work. Especially LZO...

Comment (maltesander) [reply docs/modules/hbase/pages/usage-guide/compression.adoc:7]: Should be fixed now here https://github.com/stackabletech/hbase-operator/pull/417

**Comment by lfrancke** [comment]
I wonder if this should be its own page or on the same page as the PR from Sigi just now: https://github.com/stackabletech/hbase-operator/pull/416 Opinions?

**Review by maltesander — APPROVED** [review]
LGTM!
I think it does not fit into cluster operations, so its own file is fine with me.

**Pull request merged**